org.bitbucket.mystery_keeper.VapourSynthEditor.appdata.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>org.bitbucket.mystery_keeper.VapourSynthEditor</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>MIT</project_license>
+	<name>VapourSynth Editor</name>
+	<summary>Editor for VapourSynth scripts.</summary>
+	<description>
+		<p>VapourSynth Editor is a cross-platform editor for VapourSynth scripts.</p>
+	</description>
+	<launchable type="desktop-id">org.bitbucket.mystery_keeper.VapourSynthEditor.desktop</launchable>
+	<screenshots>
+		<screenshot type="default">
+			<image type="source">screen1.png</image>
```

Comment:
I won't have URL addresses until they're merged into an own Flathub-hosted repo as they're on github right now. This will of course be fixed if the pull is accepted.

Reply:
You can consider putting these screenshots on [Debian Screenshots](https://screenshots.debian.net).

Reply:
Never heard of it before, but will make sure to use it from now on :)

Thanks!

Reply:
Debian Screenshots replied and notified me that they only host images of software in the Debian ecosystem, which this application is not.